src/parser/jobs/dnc/modules/Procs.js

```diff
@@ -29,28 +40,86 @@ export default class Procs extends Module {
 		[STATUSES.FLOURISHING_WINDMILL.id]: 0,
 	}
 
+	_buffWindows = {
```

Comment:
consider generating initial states lazily so you don't need to write them all out one by one.